### DISCUSSION TITLE
Android agent app no more in play store

### DISCUSSION THREAD

**Discussion by tradexsrl** [Ylianst/MeshCentral#6480] [Q&A] [answered]
Hi , until 2 weeks ago i was able to download meshcentral agent , today i tried to do it again for a new smartphone but i wasn't able to find it ....have you pulled the app from the store ?

**Comment by si458** [comment]
oh dear yes it seems to of vanished from the google play store!
i will message @Ylianst and find out what happened!

**Reply by si458** [reply, marked as answer]
@tradexsrl should be bk on the play store again.
google wanted verification information again @Ylianst said

**Reply by tradexsrl** [reply]
yes is back again , thank's
